api/v1/controllers/UploadController.js

```diff
@@ -0,0 +1,129 @@
+/* jshint esversion: 6 */
+
+// NOTE this controller is mostly example code that demonstrates how to use
+// the storage service to permit the uploading of resumes to the API (1 per user).
+// it is expected that this code will be refactored into the registration service
+// after being merged
+
+var bodyParser = require('body-parser');
+var ExpressRouter = require('express').Router;
+var _Promise = require('bluebird');
+
+var errors = require('../errors');
+var middleware = require('../middleware');
+var services = require('../services');
+var utils = require('../utils');
+
+var Upload = require('../models/Upload');
+var User = require('../models/User');
+
+const UPLOAD_ALREADY_PRESENT = "An upload has already been associated with this user";
+
+const RESUME_UPLOAD_LIMIT = '2mb';
```

Comment:
Did you mean to use the [configuration the user put in](https://github.com/HackIllinois/api-2017/pull/21/files#diff-5de5863d6ef8a28e7209752ed8e347e1R95)?